client/src/components/survey/SurveyPage1.jsx

```diff
@@ -108,7 +108,7 @@ const SurveyPage1 = props => {
 
   return (
     <div className={classes.root}>
-      <Grid container justify="center" alignItems="center" spacing={4}>
+      <Grid container justify="center" alignItems="center">
```

Comment:
```suggestion
      <Grid container justify="center" alignItems="center" spacing={4}>
```

I see why you're removing the spacing, but I think it looks good if we keep it and handle the [negative margin](https://material-ui.com/components/grid/#negative-margin). Replace `paddingBottom: '10em',` with `
    padding: '16px 16px 10em 16px',` in the `root` class and that should solve this issue.